README.md

```diff
@@ -0,0 +1,157 @@
+<!--
+---
+name: Azure Functions C# Timer Trigger using Azure Developer CLI
+description: This repository contains an Azure Functions timer trigger quickstart written in C# and deployed to Azure Functions Flex Consumption using the Azure Developer CLI (azd). The sample uses managed identity and a virtual network to make sure deployment is secure by default.
+page_type: sample
+products:
+- azure-functions
+- azure
+- entra-id
+urlFragment: starter-timer-trigger-csharp
+languages:
+- csharp
+- bicep
+- azdeveloper
+---
+-->
+
+# Azure Functions C# Timer Trigger using Azure Developer CLI
+
+This template repository contains an timer trigger reference sample for functions written in C# (isolated process mode) and deployed to Azure using the Azure Developer CLI (`azd`). The sample uses managed identity and a virtual network to make sure deployment is secure by default. You can opt out of a VNet being used in the sample by setting VNET_ENABLED to false in the parameters.
+
+This project is designed to run on your local computer. You can also use GitHub Codespaces:
+
+[![Open in GitHub Codespaces](https://github.com/codespaces/badge.svg)](https://github.com/codespaces/new?hide_repo_select=true&ref=main&repo=836901178)
```

Comment:
This button loads the http repo.  Suggest creating a new button (you can start in this repo, go thru codespaces feature and create new button)

Reply:
Good catch!